### DISCUSSION TITLE
async support for ITransactionScope

### DISCUSSION THREAD

**Discussion by Andriy1024** [ChilliCream/hotchocolate#5628] [Ideas]
Dear developers,
I faced the challenge to trigger some async work on the transaction completed event.
The current TransactionScope's Complete method is not async compatible. It would be nice if you consider adding async support for this case.
Thank you!

**Comment by michaelstaib** [comment]
Can you file a proper feature request on the issues. The main thing why we implemented it as it is was that System.Transactions.TransactionScope is not async.

We can update this by introducing a new `IAsyncTransactionScope`.

**Reply by Andriy1024** [reply]
Thank you for your response. 
I have copied it to the new feature request. 
https://github.com/ChilliCream/hotchocolate/discussions/5629
